app/controllers/programs_controller.rb

```diff
@@ -121,13 +121,13 @@ def set_program
     end
 
     def set_terms_and_units
-      @terms = Term.all.order(:term_start)
+      @terms = Term.sorted
       @units = Unit.where(id: current_user.unit).order(:name)
     end
 
     # Only allow a list of trusted parameters through.
     def program_params
-      params.require(:program).permit(:active, :title, :term_start, :term_end, :term_id, :subject, :catalog_number, :class_section, 
+      params.require(:program).permit(:active, :title, :term_id, :subject, :catalog_number, :class_section, 
```

Comment:
@britaumich - The new variable names are not included here. Please add them.

Reply:
@anantaUM The programs table doesn't have term's starts and end dates, it has term_id. It did have them when the table was initially created, and we did not have the terms table at that moment. But then I created a terms table and added term_id to the programs table.